distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1281,6 +1281,19 @@ A report is generated in _xls_ format.
 
 The ${branding} contains many tools to monitor system functionality, usage, and overall system health.
 
+===== Using the Log Viewer
+
+Monitor system logs with the `LogViewer`.
+
+* Navigate to the `LogViewer` at \${secure_url}/admin/logviewer
+
+Log events can be filtered by:
+
+* Log level (`ERROR`, `WARNING`, etc).
+** Logging may need to be configured to capture `DEBUG` OR `TRACE` messages.
+* Log message text.
+* Bundle generating the message.
```

Comment:
Do we need to say what log files are being pulled from in the viewer? I could see the case either way. On one side you just want to see logs. On the other it might be helpful if you need to go to the actual log file for some reason. I thought this was configurable as well. @garrettfreibott is that accurate?